backend/socialdist/urls.py

```diff
@@ -28,6 +28,10 @@
          FollowerViewSet.as_view({'get': 'list'})),
     path('author/<str:author_id>/followers/<str:foreign_author_id>/',
          FollowerViewSet.as_view({'get': 'retrieve'})),
+    path('author/<str:author_id>/posts/<str:post_id>/comments',
+        CommentViewSet.as_view({'get': 'list'})),
+    path('author/<str:author_id>/posts/<str:post_id>/comments/<str:comment_id',
+        CommentViewSet.as_view({'get': 'retrieve'})),
```

Comment:
If you name a function different from the HTTP method, you should define it within `as_view({})`.